.Lib9c.Plugin/PluginActionEvaluator.cs

```diff
@@ -18,7 +20,8 @@ public PluginActionEvaluator(IPluginKeyValueStore keyValueStore)
         {
             var stateStore = new TrieStateStore(new WrappedKeyValueStore(keyValueStore));
             _actionEvaluator = new ActionEvaluator(
-                _ => new RewardGold(),
```

Comment:
Is the `RewardGold` action no longer used? I'm asking because I'm not sure. 🤔

Reply:
Yes. In PoS, gold distribution will be executed by PoS logic. Missing logic will be moved to block action later.